packages/@aws-cdk-containers/ecs-service-extensions/lib/environment.ts

```diff
@@ -3,6 +3,10 @@ import * as ecs from '@aws-cdk/aws-ecs';
 import * as cdk from '@aws-cdk/core';
 import { EnvironmentCapacityType } from './extensions/extension-interfaces';
 
+// keep this import separate from other imports to reduce chance for merge conflicts with v2-main
+// eslint-disable-next-line no-duplicate-imports, import/order
+import { Construct } from '@aws-cdk/core';
+
```

Comment:
Nice! This should significantly reduce merge conflicts.

This is what I had originally suggested here - https://github.com/aws/aws-cdk/pull/12054#pullrequestreview-555364422